www/package.json

```diff
@@ -4,7 +4,7 @@
   "repository": "https://gerrit.wikimedia.org/r/apps/ios/wikipedia",
   "version": "0.1.0",
   "dependencies": {
-       "applib": "0.1.2"
+       "applib": "1.0.0"
```

Comment:
I updated `applib` version to `1.0.0`, then ran `make grunt`, which caused `Wikipedia/assets/index.js` to be rebuilt with upstream changes.